CHANGES.rst

```diff
@@ -0,0 +1,53 @@
+####################
+crate_ruby changelog
+####################
+
+
+in progress
+===========
+Modernize repository and testing environment
```

Comment:
I'd remove this line or otherwise it should be a list item and the other sublist item. But I'd prefer to just skip it.

Reply:
In turn, I recognized that we don't want to share all of those details at all with downstream users as they are mere internal details? I will probably amend this section and just say.

```
in progress
===========
- Modernize repository and testing environment
```

leaving out all of the other items. Do you agree?

Reply:
yeah right. question is then if we should add a changes entry at all as everything is not user facing. maybe leave it out completely?